src/MeetingsSDKAdapter.js

```diff
@@ -355,8 +380,17 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         map(() => this.meetings[ID])
       );
 
+      // Listen to unmute event to return the meeting object
+      const meetingWithLocalUnmuteEvents$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_UNMUTE).pipe(
+        map(() => this.meetings[ID])
+      );
+
       // Merge all event observables to update the existing meeting object simultaneously
-      const meetingsWithEvents$ = merge(meetingWithReadyEvent$, meetingWithLocalMuteEvents$);
+      const meetingsWithEvents$ = merge(
+        meetingWithReadyEvent$,
```

Comment:
Could you rename this to `meetingWithMediaReadyEvent$`? I know it isn’t a part of this PR but it keeps confusing me